### PR TITLE
API Guide shortcuts

### PR DESCRIPTION
**Pull request by angiejones**
added API Reference links to Web5 and SSI categories issue #619

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: site/docs/ssi/create-credentials.md
Comment (angiejones) [inline site/docs/ssi/create-credentials.md:1]: ignore this file. this commit shouldn't be part of this PR. rebase being a bully

File: site/docs/ssi/credential-lifecycle.md
Comment (angiejones) [inline site/docs/ssi/credential-lifecycle.md:1]: ignore this file. this commit shouldn't be part of this PR. rebase being a bully

File: site/docs/ssi/create-did.md
Comment (angiejones) [inline site/docs/ssi/create-did.md:1]: ignore this file. this commit shouldn't be part of this PR. rebase being a bully
